ts/private/ts_validate_options.bzl

```diff
@@ -51,20 +59,18 @@ To disable this check, set the validate attribute to False:
         resolve_json_module = ctx.attr.resolve_json_module,
         source_map = ctx.attr.source_map,
         incremental = ctx.attr.incremental,
-        ts_build_info_file = ctx.attr.ts_build_info_file,
+        # ts_build_info_file = ctx.attr.ts_build_info_file,
```

Comment:
?

Reply:
err yeah... there's no such attribute on ts_project so I think it was a mistake here

Reply:
oh I see, there's a string-typed `ts_build_info_file` and then `ts_project#buildinfo_out` was a slightly different label.

Fixed this the simple (and conservative) way to restore how it worked before by adding another attribute.